vnext/Microsoft.ReactNative/Modules/Animated/FacadeType.h

```diff
@@ -41,6 +44,6 @@ inline FacadeType StringToFacadeType(const std::string &value) noexcept {
   if (value == "perspective")
     return FacadeType::Perspective;
 
-  assert(value == "progress");
-  return FacadeType::Progress;
```

Comment:
with this change we won't ever use the progress facade

Reply:
Sure, I didn't notice that `progress` is returned only here, without previous `if`.
I brought back the `FacadeType::Progress` handling.